### PR TITLE
reaver-wps-t6x: 1.5.2 -> 1.6.3

### PR DESCRIPTION
**Pull request by nico202**

### CODE REVIEW

File: pkgs/tools/networking/reaver-wps-t6x/default.nix [hunk 1]
```diff
@@ -1,33 +1,24 @@
...
-    sourceRoot=$(echo */src)
-  '';
-
-  configureFlags = "--sysconfdir=${confdir}";
+  sourceRoot = "reaver-wps-fork-t6x-v${version}-src/src";
 
   installPhase = ''
-    mkdir -p $out/{bin,etc}
-    cp reaver.db $out/etc/
+    mkdir -p $out/bin
```
Comment (c0bw3b) [inline pkgs/tools/networking/reaver-wps-t6x/default.nix]: Isn't this redundant with the previous `mkdir -p $out/{bin,etc}` ?

File: pkgs/tools/networking/reaver-wps-t6x/default.nix [hunk 2]
```diff
@@ -1,33 +1,24 @@
...
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ libpcap sqlite pixiewps ];
+  buildInputs = [ libpcap pixiewps ];
 
-  setSourceRoot = ''
-    sourceRoot=$(echo */src)
-  '';
-
-  configureFlags = "--sysconfdir=${confdir}";
+  sourceRoot = "reaver-wps-fork-t6x-v${version}-src/src";
```
Comment (pbogdan) [inline pkgs/tools/networking/reaver-wps-t6x/default.nix]: Why did you change the `sourceRoot` here? As is the package won't build because of this change.

**Comment by orivej** [comment]
You have deleted the issue template, reverted a necessary sourceRoot change, and removed dependency on sqlite without explanation.  Could you make merging this harder?

**Comment by nico202** [comment]
Sorry, it's the wrong commit, this was a wip, I did a rebase/fixup but this has not been pulled. I'll tag you when I'll sort out what's the problem with my git

**Comment by orivej** [comment]
OK, thank you!

**Comment by nico202** [comment]
@orivej @pbogdan ok sorry for the noise. Now it should be ok.
The sqlite dependency has been removed here https://github.com/t6x/reaver-wps-fork-t6x/commit/46ccf2e268f74ca12c250157ae6017732e818f41

**Review by orivej — APPROVED** [review]

**Comment by c0bw3b** [comment]
Builds and runs

```shell
Result in /tmp/nox-review-_j6wgu5f
total 4
lrwxrwxrwx 1 demo users 64  8 nov.  18:48 result -> /nix/store/wcbrwimq0xkn79gpx77wmc65mg96b3mq-reaver-wps-t6x-1.6.3

$ /nix/store/wcbrwimq0xkn79gpx77wmc65mg96b3mq-reaver-wps-t6x-1.6.3/bin/reaver --help
Reaver v1.6.3 WiFi Protected Setup Attack Tool
Copyright (c) 2011, Tactical Network Solutions, Craig Heffner <cheffner@tacnetsol.com>

```
